### ISSUE TITLE
Ore generator

### ISSUE THREAD

**Issue by DevCreationAI** [PotatoeTrainYT/PluginRequests#3]
# Want a plugin made? Request it here!

# Plugin name
name: Ore Generator

# Plugin API (3.0.0-ALPHA10 for latest version)
api: 3.0.0-ALPHA10

# Details description:
description: Re-Generates Ore after they have been mined. (Re-spawns an ore after mined)

# Commands:
/gore edit <level> enable:disable (Enable means OreGenerator activates on the specific level or, world, that is typed.

The Config.yml can have options to enable or disable ore generation in a world easily, it can be used to change the item or, ore, that must be regenerated after mined.
# heal, heals player, need perm heal.me

**Comment by xXNiceAssasinl0** [comment]
Like cobblestone generator?

**Comment by PotatoeTrainYT** [comment]
i may not do this idk probs will

**Comment by xXNiceAssasinl0** [comment]
Like cobblestone generator?

**Comment by xXNiceAssasinl0** [comment]
Like cobblestone generator? @DevCreationAI

**Comment by DevCreationAI** [comment]
Yes but for other ores too, this is an example of the Cosmic Prisons mines.

**Comment by xXNiceAssasinl0** [comment]
:/

**Comment by PotatoeTrainYT** [comment]
ive made this before but idk if i wanna realease public for now as it may mess up my plans for the future

**Comment by xXNiceAssasinl0** [comment]
Ok
